Automattic/wp-calypso
# jsconfig: Add further jsconfig excludes according to latest Microsoft suggestions.

I'm still seeing "exclude large folders" warnings in VS Code with our current `jsconfig.json`. I noticed in [recent Microsoft advice](https://github.com/Microsoft/vscode/issues/31711#issuecomment-344795394) this added `node_modules` rule; I'm the first to admit I don't see how the addition of this rule would make a difference in Calypso, but it takes care of my warnings. ¯\_(ツ)_/¯

**Testing**
* Do a fresh checkout of Calypso.
* Open VS Code within Calypso, and open a file or two; you should see a prompt to exclude large files.
* Switch to this branch, and do the same; you shouldn't see the warning.
